Artifacts/AUTHORING.md

```diff
@@ -0,0 +1,144 @@
+# Authoring Artifacts
+
+An Artifact consists of an artifact definition file and other script files that are stored
+in a folder in a git repository. Artifact definition file consists of JSON and expressions
+which you can use to specify what you want to install on a machine. For example, you can 
+define the name of artifact, command to run and parameters that are made available when 
+command is run. You can refer to other script files within artifact definition file using 
+their name.
+ 
+## Concepts
+-------------------------------------------------------------------------------
+### __Artifact definition file format__
+
+The following example shows the sections that make up the basic structure of a definition 
+file.
+
+    {
+      "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+      "title": "",
+      "description": "",
+      "iconUri": "",
+      "targetOsType": "",
+      "parameters": {
+        "<parameterName>": {
+          "type": "",
+          "displayName": "",
+          "description": ""
+        }
+      },
+      "runCommand": {
+        "commandToExecute": ""
+      }
+    }
+
+
+|Element Name |Required |Description
+|-------------|:--------|:-----------
+|$schema      |No       |Location of the JSON schema file that helps in testing the validity of the definition file.
+|title        |Yes      |Name of the Artifact that is displayed in Lab
+|description  |Yes      |Description of the Artifact that is displayed in Lab
+|iconUri      |No       |Uri of the icon that will be displayed in the Lab
```

Comment:
remove 'the' to be consistent.

Reply:
Instead, I added `the` where needed to make it read better.